lib/urbanopt/scenario/scenario_post_processor_opendss.rb

```diff
@@ -199,13 +206,15 @@ def save_transformers_reports
             transformer_report.power_distribution.under_voltage_hours = under_voltage_hrs
             transformer_report.power_distribution.nominal_capacity = nominal_capacity
             transformer_report.power_distribution.reactance_resistance_ratio = r_r_ratio
+            transformer_report.power_distribution.tx_incoming_voltage = t_res[t_key][:incoming_voltage]
+            transformer_report.power_distribution.tx_outgoing_voltage = t_res[t_key][:outgoing_voltage]
```

Comment:
@kflemin Are there better output names than `tx_incoming_voltage` and `tx_outgoing_voltage` in the report? If so, we'll need to change both here and in https://github.com/urbanopt/urbanopt-reporting-gem/pull/138